README.md

```diff
@@ -141,14 +153,17 @@ import { StandardMerkleTree } from "@openzeppelin/merkle-tree";
 ### `StandardMerkleTree.of`
 
 ```typescript
-const tree = StandardMerkleTree.of([[alice, '100'], [bob, '200']], ['address', 'uint']);
+const tree = StandardMerkleTree.of([[alice, '100'], [bob, '200']], ['address', 'uint'], { sortLeaves: true });
```

Comment:
If `true` is the default, I don't think we should add this option here. Perhaps a section with the `MerkleTreeOptions` docs would be better.

```suggestion
const tree = StandardMerkleTree.of([[alice, '100'], [bob, '200']], ['address', 'uint'], options);

#### Options

| Option                  | Description                                                                                                                             | Default |
| ----------------- | ------------------------------------------------------------------------------------ | ------- |
| `sortLeaves`       |   Enable or disable sorted leaves. Sorting is strongly recommended for multiproofs.    | `true`   |
```

Reply:
we cannot add a section like that in the middle of a code block. Let me know if the current version works for you.